domains/vijay.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Vijay's profile",
+    "repo": "https://github.com/hack-e-d/profile.github.io",
+    "owner": {
+        "username": "hack-e-d",
+        "email": "pvvijayraj1998@gmail.com",
+    },
+    "record": {
+        "CNAME": "hack-e-d.github.io/profile/"

```

Comment:
`CNAME` record cannot contain any slashes.
Check the [docs](https://docs.is-a.dev/github_pages/) for information about setting up a subdomain from is-a.dev with GitHub pages.

Reply:
changed with slash removal